src/node-lib/curriculum-api-2023/fixtures/quizElements.new.fixture.ts

```diff
@@ -354,3 +355,413 @@ export const quizQuestionsNoImages: QuizQuestion[] = [
     order: 2,
   },
 ];
+
+export const sectionResults: LessonSectionResults = {
```

Comment:
This doesn't belong in this file. Can you make a new one called lessonSectionResults.fixture